changelog/new_merge_pull_request_459_from.md

```diff
@@ -0,0 +1 @@
+* [#462](https://github.com/rubocop/rubocop-performance/pull/462): Merge pull request #462 from corsonknowles:add_performance_use_zip_to_wrap_arrays. ([@corsonknowles][])
```

Comment:
Please write a meaningful message

Reply:
Done, will push it all up as 1 over-write.